challenge_5/python/ajschrier/FindTheDifference.py

```diff
@@ -0,0 +1,21 @@
+from collections import Counter
+
+
+class Solution():
```

Comment:
Did that to match the unit test file that was provided, I agree about the style point though